test/acceptance/ramp-buckets-regressions.test.js

```diff
@@ -53,3 +53,122 @@ describe('conditional ramp-buckets', function () {
     });
   });
 });
+
+describe('Buckets calculation', function () {
+  function createCartoCSS (numBuckets) {
+    return [
+      '#layer {',
+      '  marker-width: ramp([population], range(1, 100), quantiles(' + numBuckets + '));',
+      '  marker-fill: #EE4D5A;\n  marker-fill-opacity: 0.9;\n',
+      '  marker-allow-overlap: true; \n  marker-line-width: 1; \n',
+      '  marker-line-color: #FFFFFF; \n  marker-line-opacity: 1; \n',
+      '}'
+    ].join('\n');
+  }
+
+  var scenarios = [
+    {
+      desc: 'Buckets: 1',
+      numBuckets: 1,
+      dataSourceStats: { min_val: 1, max_val: 100, avg_val: 50 },
```

Comment:
These stats don't match with the `ramp`. For the value `[1]` the stats would look like:
```
{ min_val: 1, max_val: 1, avg_val: 1 }
```
Am I missing something?

Reply:
This stats object is meant for the whole dataset, not per bucket. So I think it makes sense as it is.